lightningd/jsonrpc.c

```diff
@@ -977,7 +977,7 @@ json_tok_address_scriptpubkey(const tal_t *cxt,
 	const char *bip173;
 
 	bool parsed;
-	bool right_network;
+	bool right_network = false;
```

Comment:
Compiler is wrong here, probably needs a comment explaining that we initialize this solely because the lint cannot figure it out.